caddyhttp/proxy/reverseproxy.go

```diff
@@ -214,6 +220,9 @@ func (p *ReverseProxy) ServeHTTP(rw http.ResponseWriter, outreq *http.Request, r
 }
 
 func (p *ReverseProxy) copyResponse(dst io.Writer, src io.Reader) {
+	buf := bufferPool.Get()
+	defer bufferPool.Put(buf)
```

Comment:
Nice - do you happen to know what the approximate/general improvement is using this buffer pool?